test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -47,6 +47,22 @@ random_generator::generate_string(std::size_t length)
     return (random_string);
 }
 
+std::string
+random_generator::generate_pseudo_random_string(std::size_t length)
```

Comment:
This should result in compression being much more effective, let me know what you think. Also it'll be a lot faster I think?